opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/SpanConverter.java

```diff
@@ -40,7 +40,8 @@ static SpanKind mapKind(@Nullable io.opencensus.trace.Span.Kind ocKind) {
     return SpanKind.INTERNAL;
   }
 
-  static Span fromOtelSpan(io.opentelemetry.api.trace.Span otSpan) {
+  @Nullable
+  static Span fromOtelSpan(@Nullable io.opentelemetry.api.trace.Span otSpan) {
     if (otSpan == null) {
       return null;
```

Comment:
Wouldn't we normally convert `null` to `Span.getInvalid()`?

Reply:
No clue since this is OpenCensus :P The annotations are correct though